templates/tab/csharp/non-sso/Components/Welcome.razor.css

```diff
@@ -0,0 +1,41 @@
+﻿.narrow {
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.page-padding {
+  padding: 4rem;
+}
+
+.welcome.page > .narrow > img {
+  margin: 0 auto;
+  display: block;
+  width: 200px;
+}
+
+.welcome.page > .narrow > .menu {
+  width: 80%;
+  justify-content: space-between;
+  margin: 4rem auto;
+  border-bottom: none;
+}
+
+  .welcome.page > .narrow > .menu > .menu-item {
```

Comment:
format this file

Reply:
I also find the format is strange here. I will fix the format in another pr.